src/extensions/step_handlers/swupdate_handler/src/swupdate_handler.cpp

```diff
@@ -143,7 +143,8 @@ ADUC_Result SWUpdateHandlerImpl::Download(const tagADUC_WorkflowData* workflowDa
     updateTypeOk = ADUC_ParseUpdateType(updateType, &updateName, &updateTypeVersion);
     if (!updateTypeOk)
     {
-        Log_Error("SWUpdate packages download failed. Unknown Handler Version (UpdateDateType:%s)", updateType);
+        Log_Error(
```

Comment:
why not output updateType in trace? seems like that's relevant?

Reply:
There's only one way the updateType can't be parsed - when it's not following the delineated path. Plus there's the option that updateType is not properly allocated/handled. Hence why I removed the printout.